coeficiente.py

```diff
@@ -0,0 +1,12 @@
+def coeficiente_rendimento(notas):
+    ''' Recebe um array com as notas do aluno e retorna o coeficiente de
+        rendimento do mesmo.
+    '''
+    soma_notas = 0
+
+    for nota in notas:
```

Comment:
A função está correta, mas como você está usando um array como entrada, poderia ser mais interessante utilizar o Numpy para calcular a média. Por exemplo:


```
# Python Program illustrating
# numpy.mean() method
import numpy as np
	
# 1D array
arr = [20, 2, 7, 1, 34]

print("arr : ", arr)
print("mean of arr : ", np.mean(arr))

```

https://www.geeksforgeeks.org/numpy-mean-in-python/